lib/hammer_cli_foreman/hosts/common_update_options.rb

```diff
@@ -25,12 +25,6 @@ def self.included(base)
         bme_options = {}
         bme_options[:default] = 'true' if base.action.to_sym == :create
 
-        bme_options[:format] = HammerCLI::Options::Normalizers::Bool.new
-        base.option "--managed", "MANAGED", " ", bme_options
-        bme_options[:format] = HammerCLI::Options::Normalizers::Bool.new
-        base.option "--build", "BUILD", " ", bme_options
```

Comment:
I agree that this change makes sense and fixes the help strings. But it also removes the default True values. While API defaults to False for build option this patch changes the behavior of the `host create` command.

As a solution you could set the default values for these options and create action in the `request_params` below.